packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -222,7 +180,16 @@ export class AtomicSmartSnippetSuggestions implements InitializableComponent {
             id={this.getRelatedQuestionId(relatedQuestion)}
           >
             {this.renderContent(relatedQuestion)}
-            {this.renderSource(relatedQuestion)}
+
+            {relatedQuestion.source ? (
+              <atomic-smart-snippet-source
+                source={relatedQuestion.source}
+                smartSnippet={undefined}
+                isSuggestion={true}
+              ></atomic-smart-snippet-source>
+            ) : (
+              []
+            )}
```

Comment:
Alternative pattern common with JSX
```suggestion
            {relatedQuestion.source && (
              <atomic-smart-snippet-source
                source={relatedQuestion.source}
                smartSnippet={undefined}
                isSuggestion={true}
              ></atomic-smart-snippet-source>
            )}
```